sources/packages/backend/apps/db-migrations/src/migrations/1704480832432-DropShowYearOfStudyColumn.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class DropShowYearOfStudyColumn1704480832432
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Not a part of the PR. Noticed that we have been using public access specifier explicitly in migrations.

Reply:
It is the boilerplate code auto-generated by the Typeorm migration since day one. For this specific situation I did not believe that we should spend time. Removing the `public` does not consume time but having it as part of the review and asking for changes from today till the end of the project will be an unnecessary time consumption IMO.
Unless we can change the way that the code is auto-generated 😉